setup.py

```diff
@@ -117,7 +76,7 @@
     keywords="qiskit sdk quantum",
     packages=find_packages(exclude=["test*"]),
     install_requires=REQUIREMENTS,
-    setup_requires=["Cython>=0.27.1"],
+    setup_requires=["setuptools-rust"],
```

Comment:
I should have just removed this since we're relying on pyproject.toml for this and this will just use an egg/easy_install install too late to be useful anyway

```suggestion
```